### PR TITLE
add a cronjob crud example

### PR DESCRIPTION
**Pull request by xg0719**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
show people basic cronjob cases
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: examples/cronjob_crud.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+def create_namespaced_cron_job(namespace='default', body=None):
+    cronjob_json = body
+    if body is None:
+        print('body is required!')
+        exit(0)
+    name = body['metadata']['name']
+    if judge_crontab_exists(namespace, name):
+        print(f'{name} exists, please do not repeat!')
+    else:
+        v1 = client.BatchV1beta1Api()
```
Comment (yliaog) [inline examples/cronjob_crud.py:21]: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

Please use batch/v1

**Review by xg0719 — COMMENTED** [review]
change BatchV1beta1Api to BatchV1Api

File: examples/cronjob_crud.py (hunk 1 above)
Comment (xg0719) [reply examples/cronjob_crud.py]: thanks, I have used batch/v1 in the new commit

**Pull request merged**